CMakeLists.txt

```diff
@@ -201,6 +199,10 @@ if(WITH_GOLANG)
 endif(WITH_GOLANG)
 
 set(PADDLE_PYTHON_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/python/build")
+
+SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O3 -g -DNDEBUG")
+SET(CMAKE_C_FLAGS_RELWITHDEBINFO "-O3 -g -DNDEBUG")
```

Comment:
Why does it not work when setting these on line 19 and line 20?

Reply:
I am not sure. However, we are setting `FLAGS` during `flags.cmake` and other cmake files. `CMAKE_CXX_FLAGS_RELWITHDEBINFO` could be overwritten if we set it in the beginning of `CMakelists.txt`. However, we can set `CMAKE_CXX_FLAGS_RELWITHDEBINFO` just before include the `paddle` directory.